doc/api/http.md

```diff
@@ -1169,9 +1169,9 @@ added: v0.0.2
 -->
 
 * {boolean}
-
-Boolean value that indicates whether the response has completed. Starts
-as `false`. After [`response.end()`][] executes, the value will be `true`.
+*
```

Comment:
Not sure how this will be rendered in site HTML, but in GitHub, we have an empty list item with this line break.